tests/results/DiffTest.py

```diff
@@ -626,3 +626,21 @@ def test_delete(self):
         self.uut.delete = True
         self.assertEqual(self.uut.modified, [])
         self.uut.delete = False
+
+    def test_linebreaks(self):
+        diff = Diff(['1'])
+        lines = ['1', '2', '3']
+        appended = diff._add_linebreaks(lines)
+        skip_eof = diff._generate_linebreaks(lines)
+        empty = diff._add_linebreaks([])
+        empty_skip = diff._generate_linebreaks([])
```

Comment:
you don't need the diff instance, access the staticmethods from `Diff` instead of `diff` :+1: